Xgain: 0.99983035110113183
YtoX: -0.0019324595533956692
ZtoX: 0.0021761195866141733
Xofs: 0.029275408567093174
XtoY: -0.0010318305548720473
Ygain: 0.98289525790477361
ZtoY: -0.011297271007627953
Yofs: -0.04464489393659777
XtoZ: 0.0013067290538877952
YtoZ: 0.051459845595472439
Zgain: 1.0178102508304625
Zofs: -0.0083763943569553807
gyro_x_offset: -6.0314960629921259
gyro_y_offset: 5.8188976377952759
gyro_z_offset: 6.8582677165354333